DuckDuckGo/NetworkProtectionTunnelController.swift

```diff
@@ -184,10 +223,8 @@ final class NetworkProtectionTunnelController: TunnelController {
         }
     }
 
-    /// Reloads the tunnel manager from preferences.
-    ///
-    private func reloadTunnelManager() {
-        internalTunnelManager = nil
+    private func clearInternalManager() {
+        internalManager = nil
     }
```

Comment:
Renamed this function since `reload` was misleading - `clearInternalManager` is the function name on macOS so it's consistent now.